### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by rbehjati**
Updates the README with a high-level description of the project and moves the documentation about the use of the command line tools to `cmd`.

Will have to merge this after #130 as the new documentation references the documentation added in #130.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,105 @@
-# Transparent Release of Binaries
+# The Transparent Release Project
+
+Transparent Release (not really the best name for the project) builds on the idea of binary
```
Comment (mariaschett) [inline README.md:3]: "(not really the best name for the project)" --- I'd drop that. It is a bit confusing/discouraging as a first point. If you want to keep it: Can we write the history instead on why the name developed?

File: README.md [hunk 2]
```diff
@@ -1,87 +1,105 @@
...
+transparency and expands in a few directions, as described below. 
+
+## Release Transparency
+
+The goal of release transparency is to generate and transparently publish ideally-verifiable metadata about a released binary. We refer to this kind of metadata as the endorsement statement. It can be used in scenarios where the identity of a binary must be verified before use. One prominent use case is remote attestation. See Oak’s remote attestation protocol for instance.
```
Comment (mariaschett) [inline README.md:8]: ideally-verifiable -> ideally verifiable

Does it make sense to link [Oak's remote attestation protocol](https://github.com/project-oak/oak/tree/main/remote_attestation)

File: README.md [hunk 3]
```diff
@@ -1,87 +1,105 @@
...
+
+The life cycle of an endorsement statement has three phases as shown in the diagram below:
+
+![The journey of a software binary](docs/images/release-transparency.png)
+
+### The pre-release phase
+In this phase, a trusted builder, for instance [the generic slsa-generator GitHub workflow](https://github.com/slsa-framework/slsa-github-generator/blob/de4491844e9be4184f786666af40f5b1b8e7ddc0/internal/builders/generic/README.md),
+is used to generate the binary and a signed SLSA provenance statement about it. A provenance
```
Comment (mariaschett) [inline README.md:16]: Suggestion to active:

In this phase a Trusted Builder, for instance ..., generates a Binary with a corresponding SLSA Provenance statement from Sources.

File: README.md [hunk 4]
```diff
@@ -1,87 +1,105 @@
...
+of the generic slsa-generator GitHub workflow, is an instance of Rekor hosted by sigstore. The
+binary is often uploaded to a storage server for future use. This can be content-addressable storage
+such as [ent](https://github.com/google/ent). 
+
+
+### The release phase
+A binary is often released when some release-readiness criteria are met. Evaluating
+release-readiness criteria often requires manual effort. As a result, triggering the release
+process is often manual as well. To transparently document the fact that a binary meets the
+release-readiness criteria and is endorsed for use, in the release phase, an endorsement statement
```
Comment (mariaschett) [inline README.md:34]: Again suggestion to active:

the (manually triggered) Endorser generates, signs, and publishes an Endorsement statement.

File: README.md [hunk 5]
```diff
@@ -1,87 +1,105 @@
...
+for use for a specific time range. An endorsement statement is a special type of claim in our
+design. For more information about the format of an endorsement statement see [the claim format](docs/claim-transparency.md#the-claim-format)
+and [this example endorsement](schema/amber-claim/v1/example.json). 
+
+Endorsement statements can be generated using a tool that we call _endorser_. Given a binary, a
+non-empty list of its provenances, and a validity time range, the endorser generates an endorsement
+statement. The release team has to separately sign and publish the endorsement statement in a
+transparency log. 
+
+Before generating the endorsement statement, the endorser performs a number of verification steps.
```
Comment (mariaschett) [inline README.md:47]: Does the endorser use the `Verifier`?

File: cmd/README.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
+# Transparent Release of Binaries
+
+The goal of transparent release is to provide an infrastructure for generating
+verifiable provenance claims about released binaries.
```
Comment (mariaschett) [inline cmd/README.md:4]: Is this goal still accurate?

**Review by mariaschett — APPROVED** [review]
Thank you and (sorry for the delay)!

Suggestion: you mention the Remote Attestation a couple of time. Would it make sense to have this as a running example throughout the README?

**Review by rbehjati — COMMENTED** [review]
Thanks for the review. 

I think it is a great idea to add a running example. Do you want to design and add an example? Also, feel free to create a tracking issue for it.

**Pull request merged**